docs/Getting-Started.md

```diff
@@ -95,7 +95,7 @@ If you want access to the WebView2 control in your app, follow these steps:
 
   iii. Customize your app's WinUI 2.x version to version 2.8.0-prerelease.210927001 or higher. See [here](https://microsoft.github.io/react-native-windows/docs/customizing-sdk-versions) for instructions. The WinUI 2.x support for WebView2 is not yet available in "stable" releases, so for now you will need to use a prerelease version.
   
-  iv. Customize your app's WebView2 version to version 1.0.1020.30. See [here](https://microsoft.github.io/react-native-windows/docs/customizing-sdk-versions) for instructions.
+  iv. You may need to specify the `Microsoft.Web.WebView2` package in your app's `packages.config` file. If this is needed, you will get a build error listing the version of the package that you needed to specify. Simply add the package to your `packages.config`, and you should be good to go.
```

Comment:
Yes! So there's a quirk with package reference where depending on if the app was created before 0.68 or after and uses community modules, you may need to specify packages in packages.config otherwise the community modules will fail to build. Added that note here in case users hit that case.